src/core/model.py

```diff
@@ -507,9 +507,7 @@ def is_processed(self) -> bool:
         return True
 
 
-def parse_str_to_date(
-    string: str, offset_time: int = 1800
-) -> Optional[datetime]:
+def parse_str_to_date(string: str, offset_time: int = 1800) -> datetime | None:
```

Comment:
why is precommit changing to this standard? isn't this Python 3.10?

Reply:
Handled by the future import at the top.

https://github.com/asottile/pyupgrade does the rewrite.